components/filter/demo/filter-tags.html

```diff
@@ -11,6 +11,11 @@
 			import '../filter-dimension-set-value.js';
 			import '../filter-tags.js';
 		</script>
+		<style>
```

Comment:
This file updates the example to one that reproduces the issue seen in Nova. It's no longer seen with the other changes in this PR, but can be seen on main with these changes and with the `width: 100%` removed from `filter-tags`